UM/Settings/SettingFunction.py

```diff
@@ -245,6 +245,8 @@ def visit_Str(self, node: ast.Str) -> None:
     def visit_Subscript(self, node: ast.Index):
         if type(node.value) == ast.Str:
             raise IllegalMethodError("Indexing on strings is not allowed")
+        if type(node.value) == ast.Constant and isinstance(node.value.value, str):
```

Comment:
To fix (typing) compatibility with Python 3.5:
``` 
       if hasattr(ast, "Constant") and type(node.value) == ast.Constant and isinstance(node.value.value, str):
```

Reply:
While this is probably sufficient from a runtime perspective, the code checker still claims it's wrong ...

Reply:
Based another branch off of this one. Fixed it there.